cupy/core/core.pxd

```diff
@@ -94,6 +94,14 @@ cdef class Indexer:
     cdef CPointer get_pointer(self)
 
 
+cdef tuple _get_concat_shape(object obj)
```

Comment:
I think all of the declarations in `.pxd` are not needed.

Reply:
Thanks. I'll remove them.